templates/registration/public_profile.html

```diff
@@ -2,33 +2,31 @@
 {% load humanize %}
 
```

Comment:
Actually nvm, I just noticed this branch was built on top of that one. This is fine.